Core/scripts/genAllCAs.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
because we want a script, which is independent from DIRAC. This script can be added to a cron and creates the files.